numpyro/distributions/util.py

```diff
@@ -1,11 +1,13 @@
 from contextlib import contextmanager
 
+import scipy.special as osp_special
+
 import jax.numpy as np
-import scipy.special as sp
-from jax import jit, lax, partial, random
+from jax import jit, lax, random
 from jax.core import Primitive
 from jax.interpreters import ad, partial_eval, xla
 from jax.numpy.lax_numpy import _promote_args_like, _promote_shapes
+from jax.util import partial
```

Comment:
`partial` is not an exposed api of jax, so it is better to import from its origin file